src/plugin.spec.ts

```diff
@@ -0,0 +1,204 @@
+import { PNG } from 'pngjs'
```

Comment:
I didn't mock node modules, because I have a feeling that is better not to mock them since you can catch more realistic use cases. We can always mock later